test/runtime/runtime-kotlin.ts

```diff
@@ -0,0 +1,34 @@
+import * as fs from 'fs';
+import * as path from 'path';
+
+// eslint-disable-next-line no-undef
+const currentDirPath = __dirname;
+const outputDir = path.resolve(
+  currentDirPath,
+  './runtime-kotlin/src/test/kotlin'
+);
+
+// Define the TypeScript code as a string template
+const typescriptCode = `
+import com.fasterxml.jackson.annotation.JsonCreator
+import com.fasterxml.jackson.annotation.JsonProperty
+
+data class Address @JsonCreator constructor(
```

Comment:
I think you are misunderstanding me.

In this file, you need to use Modelina, to generate the models, NOT manually string templating anything 🙂